pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -789,8 +948,11 @@ func (l *LibvirtDomainManager) migrateHelper(vmi *v1.VirtualMachineInstance, opt
 		if err := prepareDomainForMigration(l.virConn, dom); err != nil {
 			return fmt.Errorf("error encountered during preparing domain for migration: %v", err)
 		}
-
-		params, err = generateMigrationParams(dom, vmi, options, l.virtShareDir)
+		domSpec, err := l.getDomainSpec(dom)
+		if err != nil {
+			return fmt.Errorf("error encountered while generating migration parameters: %v", err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to get domain spec: %v", err)
```
The error message here is the same as bellow. Copy/paste issue?